service/history/mutableStateBuilder.go

```diff
@@ -245,6 +246,7 @@ func newMutableStateBuilder(
 		LastProcessedEvent: common.EmptyEventID,
 
 		StartTime:        timestamp.TimePtr(startTime),
+		ExecutionTime:    timestamp.TimePtr(startTime.Add(timestamp.DurationValue(firstWorkflowTaskBackoff))),
```

Comment:
Those times are confusing. I think current StartTime should be named ScheduledTime, and ExecutionTime should be StartTime. :)

Reply:
agreed

Reply:
Totally agree! We could easily rename them in proto, code, and ES. But they are also mapped to corresponding columns in visibility databases and renaming there is not that easy. So I had to leave it as is.